api/cloud_api/routes/Mailer.js

```diff
@@ -7,13 +7,14 @@ const {
   OK,
   BAD_REQUEST
 } = require('../../util/constants').STATUS_CODES;
+const logger = require('../../util/logger');
 const { googleApiKeys } = require('../../config/config.json');
 const { USER, ENABLED } = googleApiKeys;
 
 // Routing post /sendVerificationEmail calls the sendEmail function
 // and sends the verification email with the verification email template
 router.post('/sendVerificationEmail', async (req, res) => {
-  if(!ENABLED && process.env.NODE_ENV !== 'test') {
+  if (!ENABLED && process.env.NODE_ENV !== 'test') {
```

Comment:
The logic here needs to be fixed. If it’s not enabled or if it’s just running tests, then return 200.